apps/backend/src/modules/kobo/tests/__mocks__/getForm.ts

```diff
@@ -0,0 +1,29 @@
+import { DisasterType, DROUGHT, FLOOD, INCIDENT } from '@wfp-dmp/interfaces';
+
+import { droughtMock } from './drought';
+import { floodMock } from './flood';
+import { incidentMock } from './incident';
+
+export const getFormMock = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  province: string | undefined,
+  disasterType: DisasterType,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
why do u need to do this

Reply:
so the mock has the same signature as the real function